src/inc/Lfo.h

```diff
@@ -0,0 +1,119 @@
+#ifndef __Lfo_Hdr__
+#define __Lfo_Hdr__
+
+#include "RingBuffer.h"
+#include <cmath>
+
+class CWavetableLFO
+{
+public:
+    /*! Creates intance for the wavetable LFO
+    \param iWavetableLength wavetable length of the LFO
+    \param iSampleRate sample rate for the LFO
+    \param fFrequency frequency for the LFO in Hz
+    \param fAmplitude amplitude for the LFO
+    */
+    CWavetableLFO(int iWavetableLength, int iSampleRate, float fFrequency, float fAmplitude):
+        m_iSampleRate(iSampleRate),
+        m_dReadIdx(0)
+    {
+        double const l_dPi = 3.14159265358979323846;
+        assert(iWavetableLength > 0);
+        pCWavetableRingBuff = new CRingBuffer<float>(iWavetableLength);
```

Comment:
Does this need to be a pointer allocated on the heap? If the member variable was just a `CRingBuffer`, rather than a pointer, you could set it up in the initializer list and not have to worry about remembering to `delete` it.
(This is possible `CWavetableLFO` because it does its initialization in the constructor, unlike `CVibrato`.)